ui/app/routes/vault/cluster/metrics/index.js

```diff
@@ -0,0 +1,26 @@
+import Route from '@ember/routing/route';
+import ClusterRoute from 'vault/mixins/cluster-route';
+import { hash } from 'rsvp';
+
+export default Route.extend(ClusterRoute, {
+  model() {
+    let entitiesModel = this.store.queryRecord('entities', {}).then(response => {
+      return response.counters.entities.total;
+    });
+
+    let httpsModel = this.store.queryRecord('http-requests', {}).then(response => {
+      let reverseArray = response.counters.reverse();
+      return reverseArray[0].total;
+    });
+
+    let tokenModel = this.store.queryRecord('tokens', {}).then(response => {
+      return response.counters.service_tokens.total;
```

Comment:
I moved the tokens and entities data manipulations into a serializer.  I did not move the http-requests because for the view with the chart (that I will display in the next PR), we'll need the full array of data on this page.  Thanks for your help on this!